src/NzbDrone.Core.Test/Download/DownloadClientTests/TransmissionTests/TransmissionFixture.cs

```diff
@@ -440,17 +440,46 @@ public void should_fix_forward_slashes()
             items.First().OutputPath.Should().Be(@"C:\Downloads\Finished\transmission\" + _title);
         }
 
-        [TestCase("2.84 ()")]
-        [TestCase("2.84+ ()")]
-        [TestCase("2.84 (other info)")]
-        [TestCase("2.84 (2.84)")]
-        public void should_version_should_only_check_version_number(string version)
+        [Test]
+        public void should_fix_mixed_slashes()
```

Comment:
Oops, a test case from another PR I'm working on got committed...
It's still valid, so I rather keep it in for now.